airflow/jobs/local_task_job.py

```diff
@@ -71,6 +74,8 @@ def __init__(
         # terminate multiple times
         self.terminating = False
 
+        self._enable_task_listeners()
```

Comment:
I think should should be moved in to `_execute` instead of the constructor -- it (currently) doesn't make much difference, to anything other than tests, but currently the constructor otherwise doesn't do anything than create an object, so lets keep it that way.

Reply:
Done.